api/test/integration/conftest.py

```diff
@@ -32,13 +35,30 @@
 standalone_env_mode = 'standalone'
 cluster_env_mode = 'cluster'
 
+# parameters for docker environment start and stop
+values_build_env = {
+    'interval': 10,
+    'max_retries': 3,
+}
+
+# parameters for docker environment healthcheck
+values_env_up = {
```

Comment:
Ideally, names should be consistent between them. I think it would be better to change `values_build_env` to `values_env_build`

Reply:
Agree